airbyte-integrations/connectors/source-typeform/source_typeform/spec.json

```diff
@@ -2,37 +2,136 @@
   "documentationUrl": "https://docs.airbyte.com/integrations/sources/typeform",
   "connectionSpecification": {
     "$schema": "http://json-schema.org/draft-07/schema#",
-    "title": "Typeform Spec",
+    "title": "Source Typeform Spec",
     "type": "object",
-    "required": ["token", "start_date"],
+    "required": ["start_date"],
```

Comment:
```suggestion
    "required": ["credentials", "start_date"],
```

Reply:
added this change to spec, updated `acceptance-test-config.yml` to skip this change in `check` and `discover` CAT tests